NO-ISSUE | Создал README проекта

cxrbyratxr opened the pull request:
- [x] Заголовок МР соответствует шаблону "IOS-##### | Что сделано, по-русски, предложение в совершенном виде глагола"
- [x] Добавлено описание к МРу

## Что было сделано:
Создал README проекта

cxrbyratxr committed:
Update README.md

Filled in the basic information of the project

mightyK1ngRichard commented:
1. Изменить название МР. Отписал в тг

mightyK1ngRichard commented:
1. Изменить название МР. Отписал в тг

mightyK1ngRichard commented:
2. Глянуть описание МРа и проставить галочки проделанного

mightyK1ngRichard commented on README.md:
```diff
@@ -1,2 +1,21 @@
++ *Управление проектом* — Jira + Confluence / YouGile, Git
++ *Разработка* — SwiftUI + Firebase (ориентировочно)
++ *Дизайн* — Figma
++ *Архитектура* — MVC (от Apple) или MVVM
```
Предлагаю просто вот так

```suggestion
+ *Архитектура* — MVC/MVVM
```

mightyK1ngRichard commented on README.md:
```diff
@@ -1,2 +1,21 @@
++ Гордеева Ульяна [tg](https://t.me/ul_gord)
+### Стек
++ *Управление проектом* — Jira + Confluence / YouGile, Git
++ *Разработка* — SwiftUI + Firebase (ориентировочно)
```
Этот блок лучше разбить на 
- Вёрстка — SwiftUI + UIKit
- Данные — Firebase (ориентировочно)

cxrbyratxr committed:
NO-ISSUE | Создал README проекта

- [x] Заголовок МР соответствует шаблону "IOS-##### | Что сделано, по-русски, предложение в совершенном виде глагола"
- [x] Добавлено описание к МРу

## Что было сделано:
Создал README проекта
